src/features/timebank/timebank-api.ts

```diff
@@ -0,0 +1,110 @@
+import { WeeklyCombinedData } from "@functions/schema";
+import { Person, DailyEntriesApi, PersonsApi, DailyEntry, Timespan, PersonTotalTime } from "src/generated/client/api";
+
+/**
+ * Namespace for Timebank API
+ */
+namespace TimebankApi {
+  export const personsClient = new PersonsApi(process.env.TIMEBANK_BASE_URL);
+  export const dailyEntriesClient = new DailyEntriesApi(process.env.TIMEBANK_BASE_URL);
+
+  /**
+   * Get list of timebank users from TimeBank API
+   *
```

Comment:
missing @param comment for access token
```suggestion
   *
   * @param keycloak access token
```